specification/ai/ImageAnalysis/routes.tsp

```diff
@@ -32,7 +32,7 @@ alias SharedAnalyzeQuery = {
   The desired language for result generation (a two-letter language code).
   If this option is not specified, the default value 'en' is used (English).
   See https://aka.ms/cv-languages for a list of supported languages.
-  At the moment, only tags can be generated in none-English languages.
+  At the moment, only tags can be generated in non-English languages.
```

Comment:
The way this sentence reads, a reader might conclude that tags are generated in non-English languages but nothing else is. Is that the intended interpretation? I wonder if what is actually meant is
>  At the moment, tags can only be generated in non-English languages.

Reply:
@mikekistler On 2nd thought, I'm going to remove this sentence, since I already give the link to the public docs which includes a table of supported languages. The service may add support any time so I don't want to mention any specifics in the ref-docs.